src/js/claims-status/components/appeals-v2/Alert.jsx

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { getAlertContent } from '../../utils/appeals-v2-helpers';
 
-const Alert = ({ alert }) => {
-  const { title, description, cssClass } = getAlertContent(alert);
+const Alert = ({ title, description, displayType }) => {
+  let cssClass;
+  if (displayType === 'take_action') {
+    cssClass = 'usa-alert-warning';
+  } else if (displayType === 'info') {
+    cssClass = 'usa-alert-info';
+  }
+
   return (
     <li>
       <div className={`usa-alert ${cssClass}`}>
         <div className="usa-alert-body">
           <h4 className="usa-alert-heading">{title}</h4>
-          <p className="usa-alert-text">{description}</p>
+          <div className="usa-alert-text">{description}</div>
         </div>
       </div>
     </li>
   );
 };
 
 Alert.propTypes = {
-  alert: PropTypes.shape({
-    type: PropTypes.string.isRequired,
-    details: PropTypes.object
-  })
+  title: PropTypes.string.isRequired,
+  description: PropTypes.object.isRequired, // nullable JSX snippet
```

Comment:
If it's nullable, should it still be required? Also, does it make sense to use `PropType.element`?

Reply:
Yeah, after some investigation, I'm updating this to an optional element proptype. Not sure why I was having issues with this earlier.